packages/diffast-misc/diffast-misc.0.1/opam

```diff
@@ -0,0 +1,50 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Miscellaneous functions for Diff/AST"
+description: "Miscellaneous functions for Diff/AST"
```

Comment:
Same here, it could be useful to have a slightly more descriptive description

Reply:
We have just updated descriptions.